tests/test_cli.py

```diff
@@ -279,7 +279,7 @@ def test_run_non_ascii_file(self):
         # Make sure the default localization conditions on this "system"
         # support UTF-8 encoding.
         loc = locale.getlocale()
-        locale.setlocale(locale.LC_ALL, 'C.UTF-8')
+        locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
```

Comment:
Please don't include this is this pull-request, we can do another one if needed.

(By the way `en_US.UTF-8` isn't supposed to be present on non-English-speaking systems, and I know it isn't on Debian build system.)